src/raster/rasterband.rs

```diff
@@ -498,6 +529,43 @@ impl<'a> RasterBand<'a> {
 
         _string(str_ptr)
     }
+
+    /// Read the band mask flags for a GDAL `RasterBand`.
+    pub fn mask_flags(&self) -> Result<GdalMaskFlags> {
+        let band_mask_flags = unsafe { gdal_sys::GDALGetMaskFlags(self.c_rasterband) };
+
+        Ok(GdalMaskFlags(band_mask_flags))
+    }
+
+    /// Create a new mask band for the layer.
+    /// `shared_between_all_bands` indicates if all bands of the dataset use the same mask.
+    pub fn create_mask_band(&mut self, shared_between_all_bands: bool) -> Result<()> {
+        let flags = if shared_between_all_bands {
+            GdalMaskFlags::GMF_PER_DATASET // It is the only valid flag here.
+        } else {
+            0x00
+        };
+
+        unsafe {
+            let rv = gdal_sys::GDALCreateMaskBand(self.c_rasterband, flags);
```

Comment:
```suggestion
            let rv = unsafe { gdal_sys::GDALCreateMaskBand(self.c_rasterband, flags) };
```

to make the `unsafe` block as small as possible.